MAUI/Calendar/migration.md

```diff
@@ -734,7 +734,7 @@ this.Content = calendar;
 <tr>
 <td>{{'[MonthLabelSettings](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.MonthViewSettings.html#Syncfusion_SfCalendar_XForms_MonthViewSettings_MonthLabelSettings)'| markdownify }}</td>
 <td>Nil</td>
-<td>Not Supported.</td>
+<td>Use properties from [MonthView](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Calendar.CalendarMonthView.html#properties).</td>
 </tr>
 <tr>
 <td>{{'[PreviousMonthBackgroundColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.MonthViewSettings.html#Syncfusion_SfCalendar_XForms_MonthViewSettings_PreviousMonthBackgroundColor)'| markdownify }}</td>
```

Comment:
MonthEventParameters | Nil | Not Supported.
-- | -- | --

CHeck


[MonthEventParameters](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.MonthEventParameters.html)	Nil	Not Supported.